docs/basic_concepts.rst

```diff
@@ -0,0 +1,172 @@
+attached to the Tangle.
+
+.. note::
+    Unlike a block in a blockchain, bundles are not first-class
+    citizens in IOTA; only transactions get stored in the Tangle.
+
+    Instead, bundles must be inferred by following linked transactions
+    with the same bundle hash.
+
+Transactions in the bundle are linked together through their ``branchTransaction``
+and ``trunkTransaction`` fields, furthermore they are indexed within the bundle
+and contain a ``bundleHash`` field that is a `unique identifier for the bundle`_.
+Read more about `how bundles are structured`_.
+
+Bundles can be classified into two categories:
+
+- `Transfer bundles`_: Bundles that contain input and output transactions. A bundle
+  always has to be balanced, meaning that input transaction values should equal
+  to output transaction values.
+- `Zero-value bundles`_: Bundles that contain only zero-value transactions.
+
+For PyOTA specific implementation details on bundles, see :py:class:`Bundle`
+and :py:class:`ProposedBundle`.
+
+Now that you are familiar with seom basic IOTA concepts, it is time to explore
+how PyOTA implements these and how you can work with them.
+
+.. _official IOTA documentation site: https://docs.iota.org/docs/getting-started/0.1/introduction/overview
+.. _ternary numerical system: https://docs.iota.org/docs/getting-started/0.1/introduction/ternary
+.. _tryte alphabet: https://docs.iota.org/docs/getting-started/0.1/introduction/ternary#tryte-encoding
+.. _iota token: https://docs.iota.org/docs/getting-started/0.1/clients/token
+.. _iota network: https://docs.iota.org/docs/getting-started/0.1/network/iota-networks
+.. _unit: https://docs.iota.org/docs/getting-started/0.1/clients/token#units-of-iota-tokens
+.. _seed: https://docs.iota.org/docs/getting-started/0.1/clients/seeds
+.. _address: https://docs.iota.org/docs/getting-started/0.1/clients/addresses
+.. _spent from: https://docs.iota.org/docs/getting-started/0.1/clients/addresses#spent-addresses
+.. _structure: https://docs.iota.org/docs/getting-started/0.1/transactions/transactions#structure-of-a-transaction
+.. _input transaction: https://docs.iota.org/docs/getting-started/0.1/transactions/transactions#input-transactions
+.. _output transaction: https://docs.iota.org/docs/getting-started/0.1/transactions/transactions#output-transactions
+.. _zero-value transaction: https://docs.iota.org/docs/getting-started/0.1/transactions/transactions#zero-value-transactions
+.. _client libraries: https://docs.iota.org/docs/client-libraries/0.1/introduction/overview
+.. _how bundles are structured: https://docs.iota.org/docs/getting-started/0.1/transactions/bundles
+.. _unique identifier for the bundle: https://docs.iota.org/docs/getting-started/0.1/transactions/bundles#bundle-hash
+.. _transfer bundles: https://docs.iota.org/docs/getting-started/0.1/transactions/bundles#transfer-bundles
+.. _zero-value bundles: https://docs.iota.org/docs/getting-started/0.1/transactions/bundles#zero-value-bundle
```

Comment:
❔ Will we need to update these links every time a new version is published to `docs.iota.org`?

(e.g., `https://docs.iota.org/docs/getting-started/0.2`, `0.3`, etc.)

Reply:
I'm afraid so. However, the docs site just has been updated to the current format after months of work, I think this getting started page will stay with us for a while.